packages/spartan/buildinfo.json

```diff
@@ -4,7 +4,7 @@
     "spartan": {
       "kind": "git",
       "git": "https://github.com/dcos/spartan.git",
-      "ref": "0b5243b8032bf475fb2c62ec4f92142d701f35ad",
+      "ref": "f7b2311c71fa833c182a13781af00a387632fb1b",
```

Comment:
https://github.com/dcos/spartan/compare/0b5243b8032bf475fb2c62ec4f92142d701f35ad...f7b2311c71fa833c182a13781af00a387632fb1b looks quite specific to the single ip-blacklist/forward-zone feature described in https://github.com/dcos/dcos/pull/1536 which is properly targeted to 1.9. TL;DR: The SHA-bump looks fine.

However, I'm not happy using the `master` ref of components in a patch release. I'd prefer that we branch/version spartan, perhaps with a branch name (e.g. `1.9.1`) to match the DC/OS release it's going out with. Or at least a `1.9.x` vs. `1.10.x` so we can backport fixes to different branches for patch releases. Maybe we don't have a need to yet, if spartan development is pretty quiet, but the day will come. @nlsun @drewkerrigan